src/main/java/org/jfrog/hudson/pipeline/common/Utils.java

```diff
@@ -273,8 +273,13 @@ public String call() throws IOException {
      * @param listener - Task listener
      * @param ws       - The workspace
      */
-    public static void launch(String taskName, Launcher launcher, ArgumentListBuilder args, EnvVars env, TaskListener listener, FilePath ws) {
+    public static void launch(String taskName, Launcher nodeLauncher, ArgumentListBuilder args, EnvVars env, TaskListener listener, FilePath ws) {
         try {
+
+            Node node = ActionableHelper.getNode(nodeLauncher);
+
+            Launcher launcher = node.createLauncher(listener);
+            
```

Comment:
Remove redundant spaces
```suggestion
            Node node = ActionableHelper.getNode(nodeLauncher);            Launcher launcher = node.createLauncher(listener);
```